src/test/java/com/glovoapp/planningpoker/e2e/pages/SessionPage.java

```diff
@@ -0,0 +1,233 @@
+                userName -> true,
+                vote -> vote.getText().matches("^[0-9.]+$")
+        );
+    }
+
+    public final void allVotesTableElementsAreEmpty() {
+        allVotesTableElementsMatch(
+                userName -> userName.getText().isEmpty(),
+                vote -> vote.getText().isEmpty()
+        );
+    }
+
+    public final boolean anyUsernameInVotesTableMatches(String value) {
+        return anyVotesTableElementsMatch(
+                userName -> userName.getText().equals(value),
+                vote -> true
+        );
+    }
+    public final boolean allVotesInTableFit (String name, String value) {
+        return  allVotesTableElementsMatch(
+                userName -> userName.getText().equals(name),
+                vote -> vote.getText().equals(value)
+        );
+    }
+
+    //Returns the logical result of the function (a, b, F(a,b))
+    private boolean allVotesTableElementsMatch(final Predicate<WebElement> userNameColumnPredicate,
+                                               final Predicate<WebElement> voteColumnPredicate) {
+        return votesTableElementsMatch(userNameColumnPredicate, voteColumnPredicate, Boolean::logicalAnd);
+    }
+
+    private boolean anyVotesTableElementsMatch(final Predicate<WebElement> userNameColumnPredicate,
+                                               final Predicate<WebElement> voteColumnPredicate) {
+        return votesTableElementsMatch(userNameColumnPredicate, voteColumnPredicate, Boolean::logicalOr);
+    }
+
+    private boolean votesTableElementsMatch(final Predicate<WebElement> userNameColumnPredicate,
+                                            final Predicate<WebElement> voteColumnPredicate,
+                                            final BinaryOperator<Boolean> reductionOperator) {
+        return getTableRows().stream()
+                .map(row -> {
+                    final List<WebElement> columns = row.findElements(By.tagName("td"));
+
+                    final WebElement userNameColumn = columns.get(0);
+                    final boolean userNameColumnValid = userNameColumnPredicate.test(userNameColumn);
+
+                    final WebElement voteColumn = columns.get(1);
+                    final boolean voteColumnValid = voteColumnPredicate.test(voteColumn);
+
+                    return userNameColumnValid && voteColumnValid;
+                }).reduce(reductionOperator)
+                .orElse(true);
+    }
+
+    public void suspendJavascript(final long millis) {
+        new Thread(() -> ((JavascriptExecutor) driver).executeScript(
+                "const date = new Date();" +
+                        "var currentDate = null;" +
+                        "do { currentDate = new Date(); }" +
+                        "while (currentDate - date < " + millis + ");"
+        )).start();
+    }
+
+    public void averageEqualsExpectedResult(final WebDriver driver, final String expectedResult) {
+        System.out.println("checking!");
```

Comment:
Don't use `System.out.println` for logs; use a proper logger: `io.vertx.core.logging`.